toptal/htmlshell
# [SAT-444] fix relative path in UI wrap

[SAT-444](https://toptal-core.atlassian.net/browse/SAT-444)

---

Change UI wrap to relative path.

## How to test

1. Clone repo, open index.html
2. Check if all assets are being loaded from relative path, instead of absolute.
